assets/js/modules/reader-revenue-manager/datastore/publications.js

```diff
@@ -40,7 +41,7 @@ const fetchGetPublicationsStore = createFetchStore( {
 			MODULE_SLUG,
 			'publications',
 			{},
-			{ useCache: true }
+			{ useCache: false }
```

Comment:
Note: While this is an unrelated change, this was suggested [here](https://github.com/google/site-kit-wp/pull/9017#discussion_r1695089410). LGTM 👍